integration_test/support/session_case.ex

```diff
@@ -0,0 +1,34 @@
+defmodule Wallaby.Integration.SessionCase do
```

Comment:
Ok sorry thought there would be now this and another SessionCase, this way it seems good. Sorry :D